server/src-lib/Hasura/Events/Lib.hs

```diff
@@ -260,6 +260,8 @@ processEventQueue logger logenv httpMgr pool getSchemaCache EventEngineCtx{..} =
       case meti of
         Nothing -> do
           logQErr $ err500 Unexpected "table or event-trigger not found in schema cache"
+          liftIO . runExceptT $ Q.runTx pool (Q.RepeatableRead, Just Q.ReadWrite) $ unlockEvent e
```

Comment:
To prevent starvation of other events due to corrupt schema cache, we can set this to retry after `5 mins` otherwise it will be picked immediately.

When we have exponential back-off retry settings, we can use that as well.

Reply:
@tirumaraiselvan Can you document known scenarios where this could happen?